index.d.ts

```diff
@@ -34,18 +34,44 @@ declare namespace pug {
   export function compileFileClient(path: string, options?: Options): string;
 
   /**
+   * Compile a Pug template and render it without locals to html string.
+   */
+  export function render(template: string): string;
+  /**
    * Compile a Pug template and render it with locals to html string.
+   * @param {(Options & LocalsObject)} options Pug Options and rendering locals
```

Comment:
@felixfbecker Ok, cool - so I guess we have to duplicate the params?

I'd still love to figure out how to not duplicate things. I don't think it's that important to have docs on every single overload as long the main one you see does.